docs/index.md

```diff
@@ -200,6 +203,7 @@ git commit -a  # commit all changes
 cd ..
 cookiecutter gh:frequenz-floss/frequenz-repo-config-python \
     --directory=cookiecutter \
+    --checkout v0.4.0 \
```

Comment:
wouldn't it be better to make a branch or tag `latest_released` so we don't forget to update this in all the places?

Reply:
Maybe, but not sure, from the maintenance POV is clearly a win but from the users POV IMHO is less clear not knowing which version you are using. I think we can also use some [macros in `mkdocs`](https://mkdocs-macros-plugin.readthedocs.io/en/latest/) to have it only in one place... 

For now since it is already merged, I would leave it as is and see how annoying it is to update. My feeling is updating in many places or just one place is equally annoying and one `sed` away.